assets/ui.css

```
@@ -143,6 +143,11 @@
   padding-top: 16px;
 }
 
+.respec-save-button:link:focus {
```

```suggestion
.respec-save-button:focus {
```

No need to use `:link` as every button has one. I think other rules also should remove it.

---

link is necessary here, it won't work otherwise, tested it.

---

Ooops, that's because of base.css. Okay, let's keep it here.